holidays/countries/mauritius.py

```
@@ -0,0 +1,203 @@
+        self._add_diwali_india(tr("Divali"))
+
+
+class MU(Mauritius):
+    pass
+
+
+class MUS(Mauritius):
+    pass
+
+
+class MauritiusHinduHolidays(_CustomHinduHolidays):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)